### DISCUSSION TITLE
Fire Sprinkler Spray Pattern

### DISCUSSION THREAD

**Discussion by darwiethepang02** [firemodels/fds#13342] [Q&A] [answered]
Hello community, 

I would like to ask why does my sprinkler not create a spray pattern even when I have included the spray angle for the sprinkler. Here is the code I use and the image of my sprinkler spray. 

PROPERTIES OF WATER SPRINKLER
&SPEC ID = 'WATER VAPOR' /
&PART ID = 'my droplets', DIAMETER = 1500., SPEC_ID = 'WATER VAPOR' /
&PROP ID = 'K-11', QUANTITY = 'SPRINKLER LINK TEMPERATURE', RTI = 148., C_FACTOR = 0.7,
ACTIVATION_TEMPERATURE = 57., OFFSET = 0.10, PART_ID = 'my droplets', FLOW_RATE = 156.77,
PARTICLE_VELOCITY = 5., SPRAY_ANGLE = -50.,50., SMOKEVIEW_ID = 'sprinkler_upright' /

<img width="485" alt="Screenshot 2024-08-20 161204" src="https://github.com/user-attachments/assets/a2fbc6bf-973b-4fa3-a09a-4e1870fd9f8a">

**Comment by rmcdermo** [comment, marked as answer]
I think you just want `SPRAY_ANGLE = 0.,50.`.  If that does not work, then please attach the full input file.  Thanks

**Reply by darwiethepang02** [reply]
Hello, I have tried it and it works. Thank you.
